envoy/http/filter_factory.h

```diff
@@ -100,17 +100,24 @@ class FilterChainFactory {
   createFilterChain(FilterChainManager& manager, bool only_create_if_configured = false,
                     const FilterChainOptions& options = EmptyFilterChainOptions{}) const PURE;
 
+  enum class UpgradeAction {
```

Comment:
I think move the comments below up here in case the enum gets used multiple places